web/pom.xml

```diff
@@ -875,10 +841,211 @@
     </plugins>
   </build>
 
-  <!-- Define profiles for running configuration using -Denv=profile.id parameter
-    Profiles configuration is defined in src/main/filters.
-    -->
   <profiles>
+
+    <!-- Schema management profiles. -->
+    <profile>
+      <id>iso19115-3.2018</id>
+      <activation>
+        <property><name>!copy</name></property>
+        <file><exists>../schemas/iso19115-3.2018</exists></file>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>${project.groupId}</groupId>
+          <artifactId>schema-iso19115-3.2018</artifactId>
+          <version>${gn.schemas.version.range}</version>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>iso19115-3.2018-resources</id>
+                <phase>process-resources</phase>
+                <goals><goal>unpack</goal></goals>
+                <configuration>
+                  <encoding>UTF-8</encoding>
+                  <artifactItems>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>schema-iso19115-3.2018</artifactId>
+                      <version>${gn.schemas.version}</version>
+                      <type>jar</type>
+                      <overWrite>false</overWrite>
+                      <outputDirectory>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins</outputDirectory>
+                      <includes>plugin/**</includes>
+                      <fileMappers>
+                        <fileMapper implementation="org.codehaus.plexus.components.io.filemappers.RegExpFileMapper">
+                          <pattern>^plugin/</pattern><replacement>./</replacement>
+                        </fileMapper>
+                      </fileMappers>
+                    </artifactItem>
+                  </artifactItems>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Unpack schema resources from maven artifacts -->
+    <profile>
+      <id>unpack_schemas</id>
+      <activation> <!-- default, disabled by -Dcopy -->
+        <property><name>!copy</name></property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>unpack-schemas</id>
```

Comment:
The 2 profiles - unpack-schemas and copy-schemas - they are not grouped correctly when showing the maven toolbox in Intellij.  I see the env settings for prod/dev/inspire grouped together because they all start with env- and this makes it easier to see the options.  So I'm recommending that these be renamed to schemas-unpack and schemas-copy so that they are grouped next to each other in the maven toolbar.

Reply:
I was not sure if it would be polite to rename `copy-schemas` so I matched its naming convention. I agree that renaming both to sort into the same spot in the IDE is a nice touch.

However for these I did not want either profile used, I wanted folks to use `-Dcopy=true` or `-Dcopy=false` so only one profile enables at a time. This is a best practice for setting up either/or profiles.

I guess that is hard to manage in IntelliJ so you need to do that by hand?

Reply:
I have renamed to `schemas-unpack` and `schemas-copy`, I also named the schema profiles to sort into the same section of the list.

<img width="245" alt="image" src="https://user-images.githubusercontent.com/629681/83227578-60f38000-a139-11ea-8004-d48c4359adbc.png">

Reply:
Looks better - thank you.
It would have been nice to also rename all the related variables/properties as well to be in line with the profile name but it is fine is these cannot be changed..

Reply:
renamed the property to match.